### ISSUE TITLE
Check & Review code errors

### ISSUE THREAD

**Issue by jesuslerma** [layer5io/meshery-app-mesh#19]
**Issue Description**
Currently, the `Check & Review` code step is failing with the following issues:
```
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/client.go:24:2: a blank import should be only in a main or test package, or have a comment justifying it
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/client.go:32:6: type name will be used as istio.IstioClient by other packages, and that stutters; consider calling this Client
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/install-istio.go:24:2: exported const URLSuffix should have comment (or a comment on this block) or be unexported
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/install-istio.go:40:6: exported type APIInfo should have comment or be unexported
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/install-istio.go:46:6: exported type Asset should have comment or be unexported
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/install-istio.go:237:10: if block ends with a return statement, so drop this else and outdent its block
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/install-istio.go:290:9: if block ends with a return statement, so drop this else and outdent its block
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/istio.go:38:1: exported method IstioClient.CreateMeshInstance should have comment or be unexported
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/istio.go:373:1: comment on exported method IstioClient.ApplyOperation should be of the form "ApplyOperation ..."
##[error]/home/runner/work/meshery-app-mesh/meshery-app-mesh/app-mesh/yaml.go:65:1: exported method YAMLDecoder.Close should have comment or be unexported
##[error]Process completed with exit code 123.
```
https://github.com/layer5io/meshery-app-mesh/pull/18/checks?check_run_id=324040340

**Expected Behavior**
We hope to fix this issue by having all the checks passing.

**Comment by gunjanmimo** [comment]
@jesuslerma sir i solved this issue 
fixes #23 
https://github.com/layer5io/meshery-app-mesh/pull/23

**Comment by leecalcote** [comment]
@gunjanmimo much appreciated. Nice work.
